llvm/test/tools/llvm-split/AMDGPU/non-kernels-dependencies.ll

```diff
@@ -0,0 +1,50 @@
+; RUN: llvm-split -o %t %s -j 3 -mtriple amdgcn-amd-amdhsa
+; RUN: llvm-dis -o - %t0 | FileCheck --check-prefix=CHECK0 %s
+; RUN: llvm-dis -o - %t1 | FileCheck --check-prefix=CHECK1 %s
+; RUN: llvm-dis -o - %t2 | FileCheck --check-prefix=CHECK2 %s
+
+; 3 functions with each their own dependencies should go into 3
+; distinct partitions.
+
+; CHECK0-NOT: define
```

Comment:
Just curious, what is the purpose of `CHECK0-NOT` before the subsequent `CHECK0`s?

Reply:
To check we don't have more definitions than what we want